index.html

```diff
@@ -9,11 +9,65 @@
     <link rel="stylesheet" href="css/style.css">
     <link rel="shortcut icon" type="image/x-icon"  href="favicon.ico">
 </head>
-<body>
+<body>    
+    <header>
+        <nav class="topnav">
+            <img src="img/karma-logo.svg" alt="karma logo" 
+                width="30"
+                height="auto">
+            <ul class="nav-links">
+                <li><a href="#">Meet Karma</a></li>
+                <li><a href="#">How it Works</a></li>
+                <li><a href="#">Store</a></li>
+                <li><a href="#">Blog</a></li>
+                <li><a href="#">Help</a></li>
+                <li><a href="#">Login</a></li>
+            </ul>
+        </nav>
+    </header>
 
-    <!-- Add your HTML markup here -->
-    <!-- Remember: Use semantic HTML tags like <header>, <main>, <nav>, <footer>, <section> etc -->
-    <!-- All the images you need are in the 'img' folder -->
+<main>
+    <section>
+        <div class="hero">
+            <h1>Introducing Karma</h1>
+            <h3>Bring WiFi with you, everywhere you go.</h3>
+            <button>Learn More</button>
+        </div>
+    </section>
 
+
+    <section class="icon-cards">
+        <h2>Everyone needs a little Karma.</h2>
+        <div class="icon-container">
+            <div class="icons">
+                <img src="img/icon-devices.svg" alt="Various devices illustration" width="100" height="auto">
```

Comment:
```suggestion
                <img src="img/icon-devices.svg" alt="Various devices" width="100" height="auto">
```
We don't need to add "illustration" in the `alt` description as we know the context is an image already. Same applies to the other `img` elements below